invisible_cities/reco/hits_functions_test.py

```diff
@@ -151,3 +154,52 @@ def test_threshold_hits_all_larger_than_th(hits, th):
     hits_thresh_cor  = threshold_hits(hits, th, on_corrected = True)
     assert (h.Q  > th or h.Q == NN for h in hits_thresh    )
     assert (h.Qc > th or h.Q == NN for h in hits_thresh_cor)
+
+@dataclass
+class hits_test:
+    Ec : float
+    X  : float
+    Y  : float
+    Z  : float
+
+@dataclass
+class hitsc_test:
+    hits : list
```

Comment:
It is confusing that you ask for `Hit`type but then pass the custom `dataclass`that has some of the `Hit` attributes - ie just construct the needed `evm` type. Also maybe think of more general test: ie - large enough blob radius to encompass all hits should return energy of all hits, 0-radius or far enough center should return0s, if position of both blobs are the same the information should be repeated etc..